peano.h

```diff
@@ -10,12 +10,17 @@ typedef uint64_t morton_t;
 #define  BITS_PER_DIMENSION 21	/* for Peano-Hilbert order. Note: Maximum is 10 to fit in 32-bit integer ! */
 #define  PEANOCELLS (((peano_t)1)<<(3*BITS_PER_DIMENSION))
 
-#define DomainFac(len) ( 1.0 / (len) * (((peano_t) 1) << (BITS_PER_DIMENSION)))
+peano_t peano_hilbert_key(int x, int y, int z, int bits);
+morton_t morton_key(int x, int y, int z, int bits);
 
-#define PEANO(Pos) peano_hilbert_key((int) ((Pos[0] + All.BoxSize/2000) * DomainFac(All.BoxSize*1.001)), \
-        (int) ((Pos[1] + All.BoxSize/2000) * DomainFac(All.BoxSize*1.001)), \
-        (int) ((Pos[2] + All.BoxSize/2000) * DomainFac(All.BoxSize*1.001)), \
-        BITS_PER_DIMENSION)
+static inline peano_t PEANO(double *Pos, double BoxSize)
```

Comment:
I like keeping it a static inline function.

Reply:
We don't have a concrete case (other than precaution) to justify the factor of 1.001, do we? Either case lets note it in the code, and update when someone figures it out.

Reply:
We do not; I just kept it because that is what it always has been. I'll add a comment